### PR TITLE
mongoDB docs improvements

### PR DESCRIPTION
**Pull request by gsps1**
https://issues.cask.co/browse/CDAP-4727

### CODE REVIEW

File: mongodb-plugins/docs/MongoDB-batchsink.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ Converts a StructuredRecord into a BSONWritable and then writes it to a MongoDB
 
 Configuration
 -------------
-**connectionString:** MongoDB connection string.
\ No newline at end of file
+**connectionString:** MongoDB Connection String. *Example:'mongodb://localhost:27017/analytics.users'* 
+[Reference](http://docs.mongodb.or  g/manual/reference/connection-string)
```
Comment (gokulavasan) [inline mongodb-plugins/docs/MongoDB-batchsink.md]: remove space in org

**Comment by awholegunch** [comment]
I've made a number of changes. Please review. Otherwise LGTM.

@albertshau : Can you add a "Docs" label to this repo?

**Comment by awholegunch** [comment]
OK, now I _really_ have made a "number of changes", as two of my earlier ones got lost.

**Comment by gsps1** [comment]
okay, i rebased the commits. merging.

**Pull request merged**